lib/resources/r4/questionnaire_response.yaml

```diff
@@ -2,6 +2,19 @@
 name: QuestionnaireResponse
 field_name_base_url: https://hl7.org/implement/standards/fhir/R4/questionnaireresponse-definitions.html#QuestionnaireResponse
 fields:
+  - name: Answer Value
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.answer.value[x]
```

Comment:
its only for valueCoding answers so maybe  'Item Answer ValueCoding' and 'answer.valueCoding'. Im not sure though

Reply:
We called it a valueCoding answer in the .md, so I'm calling it that here. I changed it to answer.valueCoding, but I'm also not sure if that's what we want it as. 
Changed here: 238972cfdef64efd0724d5982fe3a4e994ea1a88 & fixed the name here to capitalize value: https://github.com/cerner/fhir.cerner.com/pull/669/commits/90d4629305fe83a34696c34438d833cfbf1e340b

Reply:
We called it a valueCoding answer in the .md, so I'm calling it that here. I changed it to answer.valueCoding, but I'm also not sure if that's what we want it as.
Changed here: 238972c & fixed the name here to capitalize value: 90d4629